pkg/api/mlflow/service/artifact/storage/s3.go

```diff
@@ -121,3 +122,23 @@ func (s S3) List(artifactURI, path string) ([]ArtifactObject, error) {
 
 	return artifactList, nil
 }
+
+// GetArtifact returns file content at the storage location.
+func (s S3) GetArtifact(artifactURI, itemPath string) (io.ReadCloser, error) {
+	bucketName, prefix, err := ExtractS3BucketAndPrefix(artifactURI)
+	if err != nil {
+		return nil, eris.Wrap(err, "error extracting bucket and prefix from provided uri")
+	}
+
+	input := &s3.GetObjectInput{
+		Bucket: aws.String(bucketName),
+		Key:    aws.String(filepath.Join(prefix, itemPath)),
+	}
+
+	resp, err := s.client.GetObject(context.TODO(), input)
+	if err != nil {
+		return nil, err
+	}
+
+	return resp.Body, nil
```

Comment:
will be nice to read data here and close response immediately.

Reply:
i disagree, could be big